colossus/src/main/scala/colossus/service/ServiceClient.scala

```diff
@@ -308,4 +306,9 @@ extends Controller[O,I](codec, ControllerConfig(config.pendingBufferSize, config
       worker ! Kill(id.get, DisconnectCause.TimedOut)
     }
   }
+
+  private def failRequest(s: SourcedRequest, exception: Throwable): Unit = {
+    errors.hit(tags = hpTags + ("type" -> exception.getClass.getName.replaceAll("[^\\w]", "")))
```

Comment:
I think we have the exact same classname sanitizing code in `ServiceServer`.  Let's try to move that somewhere common, or the very least add a `TODO` in the code along with a github issue to clean it up in a future PR.